src/components/NewToDoForm.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class NewToDoForm extends React.Component {
+  state = {
+    toDoTitle: '',
+    selectedUserId: '',
+    selectError: false,
+    inputError: false,
+  }
+
+  addTitle = (event) => {
+    this.setState({
+      toDoTitle: event.target.value,
+      inputError: false,
+    });
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    const { toDoTitle, selectedUserId } = this.state;
+
+    if (!toDoTitle && !selectedUserId) {
+      this.setState({
+        inputError: true,
+        selectError: true,
+      });
+
+      return;
+    }
+
+    if (!toDoTitle && selectedUserId) {
+      this.setState({
+        inputError: true,
+      });
+
+      return;
+    }
+
+    if (toDoTitle && !selectedUserId) {
+      this.setState({
+        selectError: true,
+      });
+
+      return;
+    }
+
+    const { onAdd, users } = this.props;
+    const newUser = users.find(user => user.id === Number(selectedUserId));
+
+    onAdd(toDoTitle, newUser);
+    this.setState({
+      toDoTitle: '',
+      selectedUserId: '',
+      selectError: false,
+      inputError: false,
+    });
+  }
+
+  handleChange = (event) => {
+    this.setState({
+      selectedUserId: event.target.value,
+      selectError: false,
+    });
+  }
+
+  render() {
+    const { toDoTitle, inputError, selectError, selectedUserId } = this.state;
+
+    return (
+      <form onSubmit={this.handleSubmit} className="ui input">
```

Comment:
Там де 2 і більше параметрів вже краще переносити
```suggestion
      <form
        onSubmit={this.handleSubmit}
        className="ui input"
      >
```